state-chain/runtime/src/lib.rs

```diff
@@ -365,7 +367,7 @@ impl pallet_cf_ingress_egress::Config<Instance1> for Runtime {
 	type FeePayment = Flip;
 	type SwapRequestHandler = Swapping;
 	type AssetWithholding = AssetBalances;
-	type FetchesTransfersLimitProvider = NoLimit;
+	type FetchesTransfersLimitProvider = EthereumLimit;
```

Comment:
Should the same limit be applied to Arbitrum @albert-llimos ?

Reply:
yes definitely

Reply:
would rename to EvmLimit then, and apply to Arbitrum too @Janislav

Reply:
Really? Last time we talked about it, we said it will be most likely different limits for Arbitrum 🤔